src/compiler/binder.ts

```diff
@@ -2283,9 +2290,7 @@ function createBinder(): (file: SourceFile, options: CompilerOptions) => void {
     function declareModuleSymbol(node: ModuleDeclaration): ModuleInstanceState {
         const state = getModuleInstanceState(node);
         const instantiated = state !== ModuleInstanceState.NonInstantiated;
-        declareSymbolAndAddToSymbolTable(node,
-            instantiated ? SymbolFlags.ValueModule : SymbolFlags.NamespaceModule,
-            instantiated ? SymbolFlags.ValueModuleExcludes : SymbolFlags.NamespaceModuleExcludes);
+        declareSymbolAndAddToSymbolTable(node, instantiated ? SymbolFlags.ValueModule : SymbolFlags.NamespaceModule, instantiated ? SymbolFlags.ValueModuleExcludes : SymbolFlags.NamespaceModuleExcludes);

```

Comment:
> "all parameters are on their own line except the first so they must have wanted a multi-line parameter list"

It only looks at the first parameter because when editing so you can quickly change the parameter lists from single line, to multi-line by just modifying the first parameter. Otherwise informing the formatter to switch between the two would be tedious.